### PR TITLE
change the `Reset` API to be called directly

### LINKED ISSUE #152: [Frontend]`resetStore` is not store any data.
I think that the `resetStore` does not store any resource data.
So it is better to call the API directly in the `ResetButton` component.

/kind cleanup

### PR DESCRIPTION
**Pull request by 196Ikuchil**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
The `Reset` function call doesn't store any data.
So, I think that `ResetStore` is not needed.

This PR removes these `store` related files and fixes to call the API directly.
#### Which issue(s) this PR fixes:

Fixes #152

#### Special notes for your reviewer:
This PR is including the contents of #163.
So, we should be merged after #163.

/label tide/merge-method-squash
/hold

### CODE REVIEW

**Review by sanposhiho — COMMENTED** [review]
Thanks!
/lgtm
/approve

**Comment by 196Ikuchil** [comment]
@sanposhiho 
I resolved the conflict. So, please approve again?

Note: #163 was merged.
/unhold

File: web/components/TopBar/ResetButton.vue [hunk 1]
```diff
@@ -89,9 +84,9 @@ export default defineComponent({
       snackbarstore.setServerInfoMessage(message);
     };
 
-    const reset = async () => {
+    const Reset = async () => {
```
Comment (sanposhiho) [inline web/components/TopBar/ResetButton.vue:87]: Rename it to `resetFn` (instead of `Reset`)

Comment (196Ikuchil) [reply web/components/TopBar/ResetButton.vue]: @sanposhiho Thanks!

**Comment by sanposhiho** [comment]
/retitle change the `Reset` API to be called directly

**Pull request merged**